provider/resources/serverless_function.go

```diff
@@ -42,8 +47,27 @@ func (*ServerlessFunction) Create(ctx p.Context, name string, input ServerlessFu
 		return name, state, nil
 	}
 
+	cloudProvider := "genezio-cloud"
+	if input.CloudProvider != nil {
+		cloudProvider = *input.CloudProvider
+	}
+
+	backendPath,err := os.Getwd()
+	if err != nil {
+		fmt.Printf("An error occurred while trying to get the current working directory %v", err)
+		return "", ServerlessFunctionState{}, err
+	}
+
+	if input.BackendPath != nil {
+		backendPath = *input.BackendPath
+	}
+
+	language := "ts"
```

Comment:
This is not true for functions because we don't compile/build functions, hence we are supporting only `js`.
```suggestion
	language := "js"
```